### ISSUE TITLE
FP S3723: should check presence of comma only on multiline contructions

### ISSUE THREAD

**Issue by vilchik-elena** [SonarSource/SonarJS#2689]
When configuring eslint [`comma-dangle`](https://eslint.org/docs/rules/comma-dangle) rules (which is actual implementation of S3723) `always-multiline` should be set for all syntax types, otherwise default `never` is used for anything which is not array or object literal:

```
foo(a, b, c,); // FP is reported here
```

Reported in[ internal thread](https://discuss.sonarsource.com/t/function-with-rest-parameters-and-rules-detecting-presence-or-lack-of-trailing-coma/8148)
